Description: Correct source GPL version declaration.
---
Origin: upstream, https://github.com/aitjcize/cppman/commit/f9b4f7baaa8877f8b2ce20adccc4dfc5906052be
Forwarded: https://github.com/aitjcize/cppman/issues/25
--- cppman-0.3.1.orig/bin/cppman
+++ cppman-0.3.1/bin/cppman
@@ -10,7 +10,7 @@
 #
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
+# the Free Software Foundation; either version 3 of the License, or
 # (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
--- cppman-0.3.1.orig/cppman/Config.py
+++ cppman-0.3.1/cppman/Config.py
@@ -9,7 +9,7 @@
 #
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
+# the Free Software Foundation; either version 3 of the License, or
 # (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
--- cppman-0.3.1.orig/cppman/Crawler.py
+++ cppman-0.3.1/cppman/Crawler.py
@@ -9,7 +9,7 @@
 #
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
+# the Free Software Foundation; either version 3 of the License, or
 # (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
--- cppman-0.3.1.orig/cppman/Environ.py
+++ cppman-0.3.1/cppman/Environ.py
@@ -9,7 +9,7 @@
 #
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
+# the Free Software Foundation; either version 3 of the License, or
 # (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
--- cppman-0.3.1.orig/cppman/Formatter.py
+++ cppman-0.3.1/cppman/Formatter.py
@@ -9,7 +9,7 @@
 #
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
+# the Free Software Foundation; either version 3 of the License, or
 # (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
--- cppman-0.3.1.orig/cppman/__init__.py
+++ cppman-0.3.1/cppman/__init__.py
@@ -7,7 +7,7 @@
 #
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
+# the Free Software Foundation; either version 3 of the License, or
 # (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
--- cppman-0.3.1.orig/cppman/cppman.py
+++ cppman-0.3.1/cppman/cppman.py
@@ -9,7 +9,7 @@
 #
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
+# the Free Software Foundation; either version 3 of the License, or
 # (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
--- cppman-0.3.1.orig/cppman/lib/cppman.vim
+++ cppman-0.3.1/cppman/lib/cppman.vim
@@ -5,7 +5,7 @@
 "
 " This program is free software; you can redistribute it and/or modify
 " it under the terms of the GNU General Public License as published by
-" the Free Software Foundation; either version 2 of the License, or
+" the Free Software Foundation; either version 3 of the License, or
 " (at your option) any later version.
 "
 " This program is distributed in the hope that it will be useful,
--- cppman-0.3.1.orig/cppman/lib/pager_less.sh
+++ cppman-0.3.1/cppman/lib/pager_less.sh
@@ -9,7 +9,7 @@
 #
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
+# the Free Software Foundation; either version 3 of the License, or
 # (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
--- cppman-0.3.1.orig/cppman/lib/pager_vim.sh
+++ cppman-0.3.1/cppman/lib/pager_vim.sh
@@ -9,7 +9,7 @@
 #
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
+# the Free Software Foundation; either version 3 of the License, or
 # (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
--- cppman-0.3.1.orig/cppman/lib/render.sh
+++ cppman-0.3.1/cppman/lib/render.sh
@@ -9,7 +9,7 @@
 #
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
+# the Free Software Foundation; either version 3 of the License, or
 # (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
